drivers/counter/counter_rtc_qmsi.c

```diff
@@ -90,17 +90,12 @@ static int rtc_qmsi_cancel_alarm(struct device *dev, u8_t chan_id)
 	return 0;
 }
 
-static int rtc_qmsi_set_top(struct device *dev, u32_t ticks,
-			     counter_top_callback_t callback,
-			     void *user_data)
+static int rtc_qmsi_set_top(struct device *dev,
+			    const struct counter_top_cfg *cfg)
 {
 	const struct counter_config_info *info = dev->config->config_info;
 
-	ARG_UNUSED(dev);
```

Comment:
Same story here.

Reply:
same story here 😄

Reply:
Right. My bad, in both stories. 😄
How could I not see this right above the line I commented? I'm getting old, I guess...